perf-tests/clusterloader2/pkg/util/util.go

```diff
@@ -245,11 +245,20 @@ func CloneMap(src map[string]interface{}) map[string]interface{} {
 }
 
 // RandomDNS1123String generates random string of a given length.
-func RandomDNS1123String(length int) string {
+func RandomDNS1123String(length int, startpos int, endpos int) string {
 	characters := []rune("abcdefghijklmnopqrstuvwxyz0123456789")
+	characters_pos := characters
+	if startpos < endpos {
```

Comment:
Here you are using condition "startpos < endpos" to check whether there are extra api servers. 
if startpos==endpos , there is no extra api server, otherwise there is.

It is not a good design for a util function. The behavior of this util is weird and hard to understand after this change.